### PR TITLE
made measures appear fully unwrapped in preview

### PR DESCRIPTION
**Pull request by Asergey91**
added unwrapped preview of features, fixed small bug with instruction type inserting more than once

### CODE REVIEW

File: app/routes/traffic-measure-concepts/index.js [hunk 1]
```diff
@@ -25,6 +25,33 @@ export default class TrafficMeasureConceptsIndexRoute extends Route {
       query['filter[label]'] = params.code;
     }
 
-    return await this.store.query('traffic-measure-concept', query);
+    const result = await this.store.query('traffic-measure-concept', query);
+
+    for (let i = 0; i < result.length; i++) {
```
Comment (nvdk) [inline app/routes/traffic-measure-concepts/index.js:30]: nitpicking, if you don't need the index, I prefer the following:
```suggestion
    for (const measure of result) {
```

File: app/routes/traffic-measure-concepts/index.js [hunk 2]
```diff
@@ -25,6 +25,33 @@ export default class TrafficMeasureConceptsIndexRoute extends Route {
...
+      const measure = result.objectAt(i);
```
Comment (nvdk) [inline app/routes/traffic-measure-concepts/index.js:31]: this assignment is no longer required if you go for the `for x of y` format
```suggestion
```

File: app/routes/traffic-measure-concepts/index.js [hunk 3]
```diff
@@ -25,6 +25,33 @@ export default class TrafficMeasureConceptsIndexRoute extends Route {
       query['filter[label]'] = params.code;
     }
 
-    return await this.store.query('traffic-measure-concept', query);
+    const result = await this.store.query('traffic-measure-concept', query);
```
Comment (nvdk) [inline app/routes/traffic-measure-concepts/index.js:28]: you may want to consider including the template here, avoids having to request it later on

File: app/routes/traffic-measure-concepts/index.js (hunk 1 above)
Comment (Asergey91) [reply app/routes/traffic-measure-concepts/index.js:30]: I dont think that works for ember data arrays/collection whatever they are

**Review by nvdk — APPROVED** [review]
didn't test, but code wise it looks ok :ok_hand:

**Comment by nvdk** [comment]
having tested this, it is too slow. I think we should precalculate this and persist it in the database so the view can be constructed faster. confused though, because I assume GN has a similar issue that I thought we resolved with the annotated field on the template.

**Pull request merged**